brigade-worker/src/job.ts

```diff
@@ -245,6 +245,9 @@ export abstract class Job {
     return this._podName;
   }
 
+  /** displayLogs controls whether logs from the job Pod will be sent to output */
+  public displayLogs : boolean = false;
```

Comment:
Is there a different variable name for following / tailing the logs?

I think one of those might be a little more intuitive if this actually follows logs (but this is just my opinion, though)

Reply:
Hm, in the ‘brig run’ context, logs are streamed/followed/sent to stdout as ‘brig run’ is executed. There is no other behavior (mainly because ‘brig run’ gets executed with a ‘follow’ pattern, worker logs are also sent directly to stdout). In any case, I’m totally open to rename ‘displayLogs’ to ‘followLogs’ or ‘streamLogs’, thoughts? Maybe ‘grabLogs’ or ‘captureLogs’? Any particular choice?

Reply:
Not a strong opinion, but `streamLogs` sounds good to me.

My mental model is that, `brig run` *follows* logs from the brigade worker. On the other hand the worker *streams* relevant logs to the worker. This change adds job pod logs to what's streamed by the worker. So, using `stream` works to me.

@dgkanatsios @radu-matei WDYT?

Btw, thanks a lot for your PR :) This had been a long-awaited feature to me!

Reply:
agreed, `streamLogs` is a great choice, pushed a commit with this change. Thanks for the comment, hope this feature proves to be useful to you!

Reply:
I know, too late, but `streamLogs` is a great name 😄